### PR TITLE
add gbenchmark

### PR DESCRIPTION
**Pull request by noob77777**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -62,7 +62,7 @@
 #######################################################################################################################
 
 # Ubuntu 20.04 ships with CMake version 3.16.3. But CI on Mac is outdated and picking up 3.15.
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.8)
```
Comment (bisakhmondal) [inline CMakeLists.txt:65]: It'd be great if you please update the comment with the mentioned version.

One thing, cmake 3.8 is way too high. Is there any release of 3.8, at all? Please confirm.
cmake 3.16 should work fine, is there any issue?

File: build_support/benchmark_CMakeLists.txt.in [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.8)
```
Comment (bisakhmondal) [inline build_support/benchmark_CMakeLists.txt.in:1]: same here.

File: src/main/biscuit.cpp [hunk 3]
```diff
@@ -2,3 +2,13 @@
...
 void biscuit::Hello() { cout << "Hello from biscuit db.\n"; }
+
+/**
+ * Benchmark testing function do not use.
+ */
+int biscuit::Radd(int x, int y) {
+  for (int i = 0; i < y; i++) {
+    x++;
+  }
+  return x;
```
Comment (bisakhmondal) [inline src/main/biscuit.cpp:13]: Noice😂

**Review by bisakhmondal — COMMENTED** [review]
Good work :+1:   :)

File: build_support/benchmark_CMakeLists.txt.in (hunk 2 above)
Comment (noob77777) [reply build_support/benchmark_CMakeLists.txt.in]: changed to 3.16 and rebased on master

Comment (bisakhmondal) [reply build_support/benchmark_CMakeLists.txt.in]: cool

**Review by bisakhmondal — APPROVED** [review]
Merging

**Pull request merged**